### PR TITLE
Updated POC styling

### PR DESCRIPTION
**Pull request by sgober**
This is a little complex because PATINA also uses the function in ShortcutUtils.js, so the StructuredDataTemplates in Shortcuts.json got more complicated to account for all the different possible conditions (underlines, hashtags, etc).  Be sure to test all the shortcuts and sign the note, as well as that the PATINA endpoint still works.

## Submitter:

**Running the Application**

- [ ] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [ ] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [ ] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [ ] This pull request describes why these changes were made
- [ ] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [ ] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [ ] Code is commented

**Tests**

- [ ] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

**Comment by sgober** [comment]
Rebased.

File: src/shortcuts/ShortcutUtils.js [hunk 1]
```diff
@@ -1,25 +1,40 @@
 import Lang from 'lodash';
 import moment from 'moment';
 
-export function createSentenceFromStructuredData(structuredPhraseTemplate, getAttributeValue, textIfNoData) {
-    let last = 0, valueName, value;
-    let start = structuredPhraseTemplate.indexOf("${"), end;
-    let result = "";
+// This function is similar to createSentenceFromStructuredData except it adds spans around the structured data so that placeholders are styled appropriately
```
Comment (jafeltra) [inline src/shortcuts/ShortcutUtils.js]: It looks like the last parameter `isSigned` is optional. Do we want to default it here?

Comment (sgober) [reply src/shortcuts/ShortcutUtils.js]: done

**Comment by jafeltra** [comment]
Dylan checked IE for me and it looks good! I pushed one more commit that resolved a warning in the console about having !== instead of !=. Everything looks good to me now!
